### ISSUE TITLE
No sus2 Chordtype available

### ISSUE THREAD

**Issue by SebastianBoldt** [AudioKit/Tonic#17]
### Description

Currently the `Chordtype`-Type only provides a `suspendedTriad` which technically is a sus4 chord because it consists of the following intervals `[.P4, .P5]`. Wouldn't it be nice to also have a `sus2` Type? 
Is there any specific reason this is missing 🤔 ?

### Proposed Solution

Splitting the `suspendedTriad` into `suspendedSecondTriad` and `suspendedFourthTriad`

```swift
case .suspendedFourth return [.P4, .P5]
case .suspendedSecond return [.M2, P5]
```

### Describe Alternatives You've Considered

Wrapping my own `Chordtype` around the `Tonic.Chordtype`

**Comment by aure** [comment]
Hi Sebastian,

Here's the difficullty with sus2 and sus4 chords. If you play the notes A, D, E, our library will identify that as a A sus4 chord, but if the sus2 case you suggests adding were added, those notes could be classified as a D sus2 chord. This ambiguity is solved by music transcribers by listening for what is the most prominent note, or the bass note, and assuming that is the root, or by looking at the context of the song. As an example, a guitarist lifting his finger off high the high note of a D chord is making a D sus2 chord, not jumping to an A sus4.

So for us, a proposed solution would have to entail some sort of logic around how we name chords with teh same spelling based on other characteristics, like perhaps in that case finding the lowest note that could be the root and deciding to name the chord off of that note. But, that feels very subjective to me, so we have not implemented anything like that.

Aure

**Comment by Matt54** [comment]
![Screenshot 2024-01-22 at 1 47 29 PM](https://github.com/AudioKit/Tonic/assets/15333030/9b62fdc3-362c-4f90-bf4f-ff4cebcabfb4)
I thought these "Potential Chords" were added to address that issue in the demo project

**Comment by aure** [comment]
I don't even recall the notion of potential chords, will have to refresh my memory.

**Comment by aure** [comment]
Thanks @SebastianBoldt for bringing up this issue, and thanks @Matt54 for reminding me we had given it some thought.

**Comment by aure** [comment]
Fixed in 1.1 https://github.com/AudioKit/Tonic/releases/tag/1.1.0
